### ISSUE TITLE
System.Runtime.InteropServices.COMException: Catastrophic failure when running tests

### ISSUE THREAD

**Issue by 0xced** [bubibubi/EntityFrameworkCore.Jet#136]
Just wanted to let everyone know that if you are getting this COM exception while running the tests, installing the [Microsoft Access Database Engine 2016 Redistributable](https://www.microsoft.com/en-us/download/confirmation.aspx?id=54920) will solve this hard to diagnose issue.

```
System.Runtime.InteropServices.COMException
Catastrophic failure
  Exception doesn't have a stacktrace
```

**Comment by ChrisJollyAU** [comment]
Did you have any other runtime version installed? I'm just guessing that the reason it failed is that there was no Access Databse Engine runtime that it could find when trying to create certain resources.

This is what I have on my PC (and yes the 2016 is from the Click-To-Run Office 365)
<img width="643" alt="image" src="https://github.com/bubibubi/EntityFrameworkCore.Jet/assets/18402102/3b32acae-cf15-4c2a-b08e-74bca4aa2867">

**Comment by 0xced** [comment]
Indeed, I had **Microsoft Access database engine 2010** installed but the 2016 version was required to overcome this error.

![image](https://github.com/bubibubi/EntityFrameworkCore.Jet/assets/51363/39ad8ae5-00f4-4e7a-bd14-6bdb9e7c84a9)
